src/components/play/layer-settings/LayerInstrumentAlt.js

```diff
@@ -0,0 +1,159 @@
+import React, { useContext } from 'react'
```

Comment:
If there's a new component "alt", did you delete the new one?

Is there a reason you didn't change the old one?

Reply:
Ali wanted us to have both the old and new inspector but we'll eventually delete the old files.